app/services/api/v1x1/service_offering.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module V1x1
+    module ServiceOffering
+
```

Comment:
Same kind of thing here, we should have `class AddToPortfolioItem < Api::V1x0::ServiceOffering::AddToPortfolioItem; end` here?

Reply:
Ruby views nested constants different than inline: https://techblog.thescore.com/2014/05/28/how-you-nest-modules-matters-in-ruby/

So none of the inherant versioned services called from the controllers would work as expected.